app/views/moderations/_suspend_user_modal.html.erb

```diff
@@ -0,0 +1,21 @@
+<div id="suspend-user-modal" class="hidden">
+  <p>Once suspended, <%= @article.username %> will not be able to comment or publish posts until their suspension is removed.</p>
+
+  <div class="crayons-field mt-4">
+    <label for="suspend_reason_userid_<%= @article.user.id %>" class="crayons-field__label">Note:</label>
+    <textarea id="suspend_reason_userid_<%= @article.user.id %>" rows="4" cols="70" placeholder="Reason for suspension" class="crayons-textfield" required></textarea>
+    <p id="suspension-reason-error" class="crayons-field__description hidden" style="color: red;">
+      You must give a suspension reason.
+    </p>
```

Comment:
```suggestion
    <textarea id="suspend_reason_userid_<%= @article.user.id %>" rows="4" cols="70" placeholder="Reason for suspension" class="crayons-textfield" required aria-describedby="suspension-reason-error"></textarea>
    <p id="suspension-reason-error" class="crayons-field__description" style="color: red;" aria-live="assertive">
    </p>
```

This relates to my suggestion noted in `suspendUserModal`, to insert the text dynamically into the UI when the error is triggered. Doing that in combination with `aria-live` and the `aria-describedby` would let this error be fully accessible to screen reader users 🙂